js/src/views/Application/Status/status.js

```diff
@@ -60,6 +60,10 @@ class Status extends Component {
   renderConsensus () {
     const { upgradeStore } = this.props;
 
+    if (!upgradeStore || !upgradeStore.consensusCapability) {
```

Comment:
Not sure how these changes fit in with the rest (as notifications), apart from being formatting related.